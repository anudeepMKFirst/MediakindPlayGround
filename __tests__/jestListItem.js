import React from 'react';
import ListItem from "./../listitem.youi.js";
import renderer from 'react-test-renderer';

it('renders correctly', () => {
	const asset = {
		id: 1,
		name: "test"
	}

  const tree = renderer
    .create(<ListItem asset={asset}/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});