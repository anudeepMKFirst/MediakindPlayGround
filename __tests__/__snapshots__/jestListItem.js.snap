// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<YiPushButtonView
  onCompositionDidLoad={[Function]}
  onPress={[Function]}
  onTimelinesLoaded={[Function]}
  testID={1}
>
  <YiImageView
    onTimelinesLoaded={[Function]}
    source={
      Object {
        "uri": "https://image.tmdb.org/t/p/w342undefined",
      }
    }
  />
</YiPushButtonView>
`;
